mos_tests/functions/os_cli.py

```diff
@@ -51,6 +52,31 @@ def execute(self, action, flags='', params='', fail_ok=False,
     def details(self, output):
         return {x['Field']: x['Value'] for x in json.loads(output)}
 
+    def details_from_table(self, output):
```

Comment:
https://github.com/openstack/tempest/blob/master/tempest/lib/cli/output_parser.py#L51 - doing the same